src/instances.cr

```diff
@@ -75,14 +75,32 @@ spawn do
         client.connect_timeout = 5.seconds
         client.read_timeout = 5.seconds
         begin
-          stats = JSON.parse(client.get("/api/v1/stats").body)
+          req = client.get("/api/v1/stats")
+          stats = JSON.parse(req.body)
+
+          api = false
+          cors = false
+          req = client.get("/api/v1/trending")
+          if req.status_code == 200
+            begin
+              if req.headers["Access-Control-Allow-Origin"] == "*"
+                cors = true
+              end
```

Comment:
you can reduce that to `cors = (req.headers["Access-Control-Allow-Origin"] == "*")`